pkg/diff/diff_test.go

```diff
@@ -75,6 +76,11 @@ func (*testDiffSuite) TestGenerateSQLs(c *C) {
 	deleteSQL = generateDML("delete", rowsData, orderKeyCols, tableInfo, "test")
 	c.Assert(replaceSQL, Equals, "REPLACE INTO `test`.`atest`(`id`,`name`,`birthday`,`update_time`,`money`) VALUES (NULL,NULL,'2018-01-01 00:00:00','10:10:10',11.1111);")
 	c.Assert(deleteSQL, Equals, "DELETE FROM `test`.`atest` WHERE `id` is NULL;")
+
+	// test value with "'"
+	rowsData["name"] = &dbutil.ColumnData{Data: []byte("a'a"), IsNull: false}
+	replaceSQL = generateDML("replace", rowsData, orderKeyCols, tableInfo, "test")
```

Comment:
is it needed to test `generateDML("delete" ...`

Reply:
done